velox/functions/sparksql/Register.cpp

```diff
@@ -98,11 +98,7 @@ void registerFunctions(const std::string& prefix) {
   exec::registerStatefulVectorFunction(
       prefix + "rlike", re2SearchSignatures(), makeRLike);
   VELOX_REGISTER_VECTOR_FUNCTION(udf_regexp_split, prefix + "split");
-
-  // Subscript operators. See ExtractValue in complexTypeExtractors.scala.
-  VELOX_REGISTER_VECTOR_FUNCTION(udf_subscript, prefix + "getarrayitem");
-  VELOX_REGISTER_VECTOR_FUNCTION(udf_subscript, prefix + "getmapvalue");
-
+  VELOX_REGISTER_VECTOR_FUNCTION(udf_subscript, prefix + "element_at");
```

Comment:
Looks like there is a duplicate registration in L54. I assume it registers Presto's version of the element_at function. We need to fix this.

```
VELOX_REGISTER_VECTOR_FUNCTION(udf_element_at, prefix + "element_at");
```

Reply:
Thanks for your suggestion. I keep the element_at registration in L54.  Because spark has same semantics.

Reply:
> spark has same semantics.

Interesting. This is nice, but Spark's documentation suggests that semantics depend on spark.sql.ansi.enabled setting. Which semantics do we have in Velox?

Reply:
Now, velox have spark.sql.ansi.enabled = false semantics.  Do I need support spark.sql.ansi.enabled = true in this PR?

Reply:
I see. Let's add a comment to the registration call to clarify that this is the semantics of spark.sql.ansi.enabled = false. Let's also update PR title and description to explain the changes clearly. If we do not need udf_subscript, then let's remove it and corresponding test.